lightning/src/ln/channel.rs

```diff
@@ -977,11 +977,14 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 
 		if value_to_b >= (dust_limit_satoshis as i64) {
 			log_trace!(self, "   ...including {} output with value {}", if local { "to_remote" } else { "to_local" }, value_to_b);
+				let static_payment_pk_hash = WPubkeyHash::hash(&if local {
+						self.their_pubkeys.as_ref().unwrap().payment_basepoint
+					} else {
+						self.local_keys.pubkeys().payment_basepoint
+					}.serialize());
```

Comment:
indentation?

Reply:
No? Its indented because its a parameter to a fn.

Reply:
My earlier comment was rather to make the expression inside `WPubkeyHash::hash()` (i.e. the payment basepoint) a local variable, which would be more readable IMHO.

Reply:
Ah, ok. Missed that. Also responded too quick to antoine's comment, the whole thing *did* have an extra indent, I'd intended that for these lines but not the top "let" line. Fixed.